cypress/e2e/shared/checks.test.ts

```diff
@@ -115,6 +116,129 @@ describe('Checks', () => {
         }
       )
     })
+
+    it('can reconfigure a threshold check', () => {
+      const every = '5m'
+      const offset = '1m'
+      const testTags: {key: string; value: string}[] = [
+        {key: 'Prima', value: 'Smith'},
+        {key: 'Astaire', value: 'Rodgers'},
+        {key: 'Gable', value: 'Lombard'},
+      ]
+      const newMsg =
```

Comment:
it's usually a good practice to initialize the variable nearer to where it's used.

Reply:
That's debatable.  When coding in C, C++ back in the 90's it was good practice to have a variable declaration/definition block, then a logic block, especially when the variables can be reused in a number of places.  By declaring variables at the start of the namespace, I know what I'm then going to be working with. However, if you insist, this can be moved.

Reply:
Absolutely! It's not 90s anymore. 3 decades later, it's usually what I said. Regardless, if you don't want to change it, ok!